### DISCUSSION TITLE
Publish Epinio on Rancher Marketplace

### DISCUSSION THREAD

**Discussion by agracey** [epinio/epinio#1183] [Ideas]
We should add Epinio to the Rancher Marketplace sooner rather than later.

I'll be setting up a call shortly with the team that maintains the catalog to see if anything additional needs to be done.

In the meantime, here are a few resources on process and requirements:

- https://github.com/rancher/charts/blob/release-v2.6/docs/developing.md#introducing-a-new-package
- https://github.com/rancher/charts/blob/release-v2.6/docs/developing.md#making-changes-to-packages
- https://confluence.suse.com/display/EN/Feature+Charts+Helm+Requirements

**Comment by jimmykarily** [comment]
@agracey converted to a discussion. Let's make it an issue when we decide to move forward with this.

**Comment by manno** [comment]
We're waiting for the UI before we do this?

**Reply by agracey** [reply]
Probably that and the consolidation to a single chart?

**Reply by agracey** [reply]
That said, we could go ahead and start some of adding the right annotations to our chart(s) and making sure they conform to the reqs

**Comment by jimmykarily** [comment]
Single chart discussion: https://github.com/epinio/epinio/discussions/1190

**Comment by jimmykarily** [comment]
We now have a single chart, should we move on with this? @agracey

**Reply by agracey** [reply]
Yes!
